wrapanapi/systems/redfish.py

```diff
@@ -90,6 +63,58 @@ def uuid(self):
         return self._ems_ref
 
 
+class RedfishServer(Server, RedfishResource):
+    state_map = {
+        'On': ServerState.ON,
+        'Off': ServerState.OFF,
+        'PoweringOn': ServerState.POWERING_ON,
+        'PoweringOff': ServerState.POWERING_OFF,
+    }
+
+    @property
+    def server_cores(self):
+        """Return the number of cores on this server."""
+        return sum([int(p.TotalCores) for p in self.raw.Processors.Members])
```

Comment:
`[]` are not needed here and needlessly create new list. `sum` can work with iterators, so
```suggestion
        return sum(int(p.TotalCores) for p in self.raw.Processors.Members)
```
is enough.

Reply:
👍